bin/wait-for-it.sh

```diff
@@ -1,182 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
I think this file was an older version of the script that we weren't using. All of the CI was using the copy in the `ci` directory, as far as I could tell.

Reply:
oops! Good catch